src/components/prescreen-info.ts

```diff
@@ -0,0 +1,17 @@
+import styled from "@emotion/styled";
+import { breakPoints, spacing } from "../constants";
+
+export const PrescreenInfo = styled("main")`
```

Comment:
i might name this PrescreenInfoWrapper since it is styling multiple types of components. It actually might be better to break this out into a few different components, IE `PrescreenInfoHeader` for the h3, `PrescreeninfoList`, `PrescreenInfoListElement`. This is because then it makes it more explicit where the styles are coming from in the file `prescreen-1.tsx`

Reply:
alternatively, it would probably be best if we had a standard UL element, a standard LI element and a standard H3 element. They would all live in /components/